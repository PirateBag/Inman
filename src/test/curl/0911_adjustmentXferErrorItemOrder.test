Try out some errors.
=== JSON ===
{ "rows" : [ { "amount" : "1.00", "itemId" : "1", "orderId" : "3", "orderType" : "PO", "effectiveDate" : "2025-0728", "adjustmentType" : "XFER", "crudAction" : "INSERT" }  ] }