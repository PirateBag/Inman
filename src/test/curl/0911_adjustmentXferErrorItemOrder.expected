{
  "responseType" : null,
  "errors" : [ {
    "key" : 0,
    "message" : "Order (3) Item 17 and Adjustment Item 1 must specify same item."
  } ],
  "data" : [ ]
}